include/openthread/dnssd_server.h

```diff
@@ -245,6 +245,21 @@ otDnssdQueryType otDnssdGetQueryTypeAndName(const otDnssdQuery *aQuery, char (*a
  */
 const otDnssdCounters *otDnssdGetCounters(otInstance *aInstance);
 
+/**
+ * Enable or disable forwarding DNS queries to platform DNS upstream API.
+ *
+ * Available when `OPENTHREAD_CONFIG_DNS_UPSTREAM_QUERY_ENABLE` is enabled.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aEnabled   A boolean to enable/disable forwarding DNS queries to upstream.
+ *
+ * @sa otPlatDnsQueryUpstreamQuery
+ * @sa otPlatDnsCancelUpstreamQueryTransaction
+ * @sa otPlatDnsOnUpstreamQueryResponse
+ *
+ */
+void otDnssdSetUpstreamQueryEnabled(otInstance *aInstance, bool aEnabled);
```

Comment:
Shall we name it `otDnssdUpstreamQuerySetEnabled` to align other `xxxSetEnabled` API names?